conf/global_params.config

```diff
@@ -1,211 +1,327 @@
+global_lowfreq_vars = false
+
+// The OPTION sets a threshold for the sequence data quality to be used for the mpileup creation
+global_minbqual = 13
+
+// The OPTION sets a minimum forward read coverage threshold. Alleles must have a forward coverage of this VALUE or higher to be considered.
+global_mincovf = 4
+
+// The OPTION sets a minimum reverse read coverage threshold. Alleles must have a reverse coverage of this VALUE or higher to be considered.
+global_mincovr = 4
+
+// The OPTION sets a minimum number of reads indicating an allele with a phred score of at least 20.
+global_minphred = 4
+
+// The OPTION sets a minimum frequency for an allele.
+global_minfreq = 75
+
+// The option sets a minimum percentage of samples with unambiguous information for position.
+global_unambig = 95
+
+// The OPTION sets a window size in which the algorithm scans for the occurrence of multiple variants within the same sample.
+// If more than one variant occurs within this window in the same sample, the positions will be excluded.
+global_window = 12
+
+// The OPTION sets a SNP distance that is used to classify samples into groups of samples, using agglomerative clustering.
+// If SNP distances between samples are less or equal this VALUE, they are grouped together.
+global_distance = 12
+
+// The OPTION sets the maximum number of CPUs to use within the pipeline.
+// You can use more than one core in order to execute the pipeline faster. 8 is the current maximum.
+global_threads = 4
+
+
+// This OPTION sets the reference genome for the read mapping.
+// The update default path is for the bioconda::mtbseq docker container.
+global_mtb_ref = "/usr/local/share/mtbseq-1.0.3-1/var/ref/"
+
+// This OPTION sets a list of known variant positions associated to drug resistance for resistance prediction.
+// The update default path is for the bioconda::mtbseq docker container.
+global_resilist = "/usr/local/share/mtbseq-1.0.3-1/var/res/MTB_Resistance_Mediating.txt"
+
+// This OPTION sets a list of interesting regions to be used for annotation of detected variants
+// The update default path is for the bioconda::mtbseq docker container.
+global_intregions = "/usr/local/share/mtbseq-1.0.3-1/var/res/MTB_Extended_Resistance_Mediating.txt"
+
+// This OPTION specifies a gene categories file to annotate essential and non-essential genes as well as repetitive regions. SNPs in repetitive regions will be excluded for phylogenetic analysis.
+// The update default path is for the bioconda::mtbseq docker container.
+global_categories = "/usr/local/share/mtbseq-1.0.3-1/var/cat/MTB_Gene_Categories.txt"
+
+// This OPTION specifies a file for base quality recalibration. The list must be in VCF format and should contain known SNPs.
+// The update default path is for the bioconda::mtbseq docker container.
+global_basecalib = "/usr/local/share/mtbseq-1.0.3-1/var/res/MTB_Base_Calibration_List.vcf"
```

Comment:
this comments are very useful as a description, and more advanced users certainly will make use of it. I liked, and it seems very readable and the name structure fit well what we discussed on our meetings and discussions about this project!

Reply:
Thanks Davi, the next step for this would be to create a `nextflow_schema.json` file -  https://help.tower.nf/pipeline-schema/overview/ so that any user on Tower would be able to make sense of these without worrying about working on the terminal or making a typo in the configuration :)

Reply:
You're going to take care of the schema? I'd like to learn about the process of generating q schema, could we do it while we connect?

Reply:
Yeah - but before that I'd appreciate if you could do rigorous testing for https://github.com/mtb-bioinformatics/mtbseq-nf/issues/31 

There might be still some global/module options which might have been missed 😓

Reply:
hmm, tomorrow on our meeting could you give me some instruction about how to proceed with this rigorous test?

Reply:
Yup, sure thing. Please do share the meeting details hmm.